src/features/cards/pages/create.js

```diff
@@ -5,7 +5,7 @@ import { compose } from "recompose"
 import { withFormik } from "formik"
 import { Col } from "@lib/styled-components-layout"
 import { Authenticated } from "@features/common"
-import { Card, Button, Input } from "@ui/atoms"
+import { Card, ButtonPrimary, Input } from "@ui/atoms"
```

Comment:
Why you have for each Button color separate component?
Why not create this kind of api for button, example: <Button color="primary" kind="outlined">Submit</Button>